src/Microsoft.VisualStudio.LanguageServices.Razor/DocumentStructureChangedEventArgs.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.VisualStudio.Text;
+
+namespace Microsoft.VisualStudio.LanguageServices.Razor
```

Comment:
I think all of these things should be in a namespace like `Microsoft.VisualStudio.LanguageServices.Razor.Editor`. We want to do feature-area partitioning for tooling